### DISCUSSION TITLE
Disabling the check determining whether or not a fire should be a soul fire

### DISCUSSION THREAD

**Discussion by QuinnCHall** [PurpurMC/Purpur#1269] [__Dismissed__] [answered]
Fire in Minecraft is able to check whether or not it's on soul soil, which allows it to become soul fire. This block is a separate block, but it can't be placed on something other then soul fire. Being able to turn off this check would be really interesting, as it would allow for commands or data packs to place soul fire wherever a player wants. 
I don't mean that all blocks should turn fire into soul fire, but having soul fire be able to be placed anywhere would be a very cool feature.

**Comment by RhythmicSys** [comment, marked as answer]
The blocks that make soul fire are set in the vanilla datapack in a file called `soul_fire_base_blocks`, if you have blocks you want to permanently make blocks create soul fire, you'd set them there, using a datapack.

Otherwise, if you want soul fire to be able to be set on blocks that also have normal fire, this request is too specialized for the scope of Purpur, and would be best done in a plugin
